src/ensembl/src/content/app/custom-download/containers/content/CustomDownloadContent.scss

```diff
@@ -29,6 +29,11 @@
   grid-column-end: 5;
```

Comment:
regarding `grid-template-columns` for `.wrapper` in the code above — i think both `calc`s can be replaced with `1fr` (`40px 1fr 10px 1fr 40px;`). Much more readable.

Reply:
> Could you please describe/screenshot the bug so that we know what's being fixed?

I have updated the description to include a screenshot

Reply:
That's done. Thanks 👍